Makefile

```diff
@@ -142,9 +142,9 @@ static-assets:
 
 test-ember:
 	@echo "--> Installing JavaScript assets"
-	@cd ui && yarn --ignore-optional
```

Comment:
previously we had this here so we didn't install husky, lint-staged, jsdoc-to-md or storybook in CI. is that not a concern anymore?

Reply:
yep, that's correct - the is was the cause of the issue with the CI builds --ignore-optional ignored a dep that was shared and so it ended up not installed properly and CI errored out. Husky was the main concern before because we didn't want everyone to get the githooks, but that's been removed so we can also remove the flag here to fix CI.